### PR TITLE
QBD: add note about separate admins

### PR DESCRIPTION
**Pull request by d-laing**
# Description

This pull request adds a note to the QuickBooks Desktop documentation.

## Type of change

Please delete options that are not relevant.

- [x] New document(s)/updating existing

## Reviews and merging

You are responsible for getting your PR merged. Address review comments promptly and **make sure to merge the PR when ready**.
Feel free to 'Enable automerge' - your PR will automatically merge when accepted and passing the build.

### CODE REVIEW

File: docs/integrations/accounting/quickbooksdesktop/installing-the-quickbooks-connector.md [hunk 1]
```diff
@@ -41,16 +41,18 @@ If you're using the Codat Link flow, the user can download the connector after t
...
-:::
+**Administrator privileges (admin rights) are required to run the connector.** When you add a new company, a user with admin rights needs to run the installer, which will display the following dialog:
 
 <img src="/img/old/420dc1e-QBDLinkNew.JPG" />
 
 The user copies their license key (the same as their _CompanyId_ in Codat) to use in the next step.
 
+::: info What if the SMB user isn't a Windows admin?
+
+If the SMB user isn't a Windows administrator on their system, a different administrator must approve the installation and then run QuickBooks Desktop using the **Run as Administrator** option. This will initiate the first link of the company. QuickBooks Desktop must then be closed for the connector to start syncing data.
```
Comment (tom-binninton-codat) [inline docs/integrations/accounting/quickbooksdesktop/installing-the-quickbooks-connector.md:52]: Quick suggestion:
> a different administrator must approve the installation and run Quickbooks Desktop using the **Run as Administrator** option during the initial link and when adding additional companies to an existing connector.
> The connector will then only be able to sync whilst Quickbooks Desktop is closed

File: docs/integrations/accounting/quickbooksdesktop/accounting-quickbooksdesktop.md [hunk 2]
```diff
@@ -110,7 +111,7 @@ Enable the QuickBooks Desktop connector in the <a className="external" href="htt
 Here's how your SMB users interact with the QBD connector.
 
 - The SMB user downloads the connector as part of the link flow.
-- Initially, the connector installer requires administrator privileges (admin rights) to run.
+- Administrator privileges (admin rights) are needed to run the connector installer. Or, a different administrator can approve the installation and run the connector.
```
Comment (mcclowes) [inline docs/integrations/accounting/quickbooksdesktop/accounting-quickbooksdesktop.md:114]: 'Alternatively, ' instead?

Comment (pzaichkina) [inline docs/integrations/accounting/quickbooksdesktop/accounting-quickbooksdesktop.md:114]: Active voice maybe?

The user needs administrator privileges (admin rights) to run the connector installer. Alternatively, a different administrator can approve the installation and run the connector.

File: docs/integrations/accounting/quickbooksdesktop/installing-the-quickbooks-connector.md [hunk 3]
```diff
@@ -41,16 +41,18 @@ If you're using the Codat Link flow, the user can download the connector after t
...
-**The connector requires administrator privileges (admin rights) to run.** Each time you add a company, a user with admin rights needs to run the installer, which will display the following window:
-
-:::
+**Administrator privileges (admin rights) are required to run the connector.** When you add a new company, a user with admin rights needs to run the installer, which will display the following dialog:
 
 <img src="/img/old/420dc1e-QBDLinkNew.JPG" />
 
 The user copies their license key (the same as their _CompanyId_ in Codat) to use in the next step.
 
+::: info What if the SMB user isn't a Windows admin?
```
Comment (pzaichkina) [inline docs/integrations/accounting/quickbooksdesktop/installing-the-quickbooks-connector.md:50]: I assume Windows is the only OS coverage we need?

File: docs/integrations/accounting/quickbooksdesktop/accounting-quickbooksdesktop.md (hunk 2 above)
Comment (d-laing) [reply docs/integrations/accounting/quickbooksdesktop/accounting-quickbooksdesktop.md:114]: Yes, that reads better.

File: docs/integrations/accounting/quickbooksdesktop/installing-the-quickbooks-connector.md (hunk 1 above)
Comment (d-laing) [reply docs/integrations/accounting/quickbooksdesktop/installing-the-quickbooks-connector.md]: Thanks @tom-binninton-codat . I've updated the note to very closely match your wording. Please sense check this sentence:

" When installed in this way, the connector is only able to sync data while Quickbooks Desktop is closed."

File: docs/integrations/accounting/quickbooksdesktop/accounting-quickbooksdesktop.md (hunk 2 above)
Comment (d-laing) [reply docs/integrations/accounting/quickbooksdesktop/accounting-quickbooksdesktop.md]: Yes, active voice works well.

File: docs/integrations/accounting/quickbooksdesktop/installing-the-quickbooks-connector.md (hunk 3 above)
Comment (d-laing) [reply docs/integrations/accounting/quickbooksdesktop/installing-the-quickbooks-connector.md]: Yes, that's all covered on the QuickBooks Desktop requirements page.

**Comment by d-laing** [comment]
In a related change, I have added a new "Limitations" heading to the _QuickBooks Desktop requirements_ page.

**Review by tom-binninton-codat — APPROVED** [review]

**Review by pzaichkina — APPROVED** [review]

**Pull request merged**